src/ui/common/src/components/layouts/NavBar.tsx

```diff
@@ -0,0 +1,219 @@
+import { faBell, faCircleUser } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  AppBar,
+  Avatar,
+  Breadcrumbs,
+  Link,
+  Menu,
+  MenuItem,
+  Toolbar,
+  Typography,
+} from '@mui/material';
+import Box from '@mui/material/Box';
+import React, { useState } from 'react';
+import { useSelector } from 'react-redux';
+import { Link as RouterLink } from 'react-router-dom';
+
+import { RootState } from '../../stores/store';
+import { theme } from '../../styles/theme/theme';
+import UserProfile from '../../utils/auth';
+import { getPathPrefix } from '../../utils/getPathPrefix';
+import {
+  NotificationLogLevel,
+  NotificationStatus,
+} from '../../utils/notifications';
+import NotificationsPopover from '../notifications/NotificationsPopover';
+import styles from './menu-sidebar-styles.module.css';
+import { MenuSidebarWidthNumber } from './menuSidebar';
+
+const pathPrefix = getPathPrefix();
+
+export class BreadcrumbLinks {
+  static readonly HOME = new BreadcrumbLinks(`${pathPrefix}/`, 'Home');
+  static readonly DATA = new BreadcrumbLinks(`${pathPrefix}/data`, 'Data');
+  static readonly INTEGRATIONS = new BreadcrumbLinks(
+    `${pathPrefix}/integrations`,
+    'Integrations'
+  );
+  static readonly WORKFLOWS = new BreadcrumbLinks(
+    `${pathPrefix}/workflows`,
+    'Workflows'
+  );
+  static readonly ACCOUNT = new BreadcrumbLinks(
+    `${pathPrefix}/account`,
+    'Account'
+  );
+  static readonly ERROR = new BreadcrumbLinks(
+    `${pathPrefix}/404`,
+    'Page Not Found'
+  );
+
+  constructor(public readonly address: string, public readonly name: any) {}
+
+  toString() {
+    return this.name;
+  }
+}
+
+/**
+ * The `NavBar` is the core sidebar that we include throughout our UI. It
+ * is pinned to the top of every page in our UI, and it includes
+ * information about the site hierarchy, notifications, and settings/accounts page.
+ */
+const NavBar: React.FC<{
+  user: UserProfile;
+  breadcrumbs: BreadcrumbLinks[];
+}> = ({ user, breadcrumbs }) => {
+  const [userPopoverAnchorEl, setUserPopoverAnchorEl] = useState(null);
+  const [anchorEl, setAnchorEl] = useState(null);
+
+  console.log(breadcrumbs);
```

Comment:
remove logs

Reply:
> I'm also wondering why we are still keeping the old layout page?
I think @agiron123 requested I keep a copy of the old layout page just in case. LMK if you think it would be fine to just delete it!

Reply:
yeah, wanted to keep things around so we can easily go back to other nav scheme if needed as a safety net for our upcoming release.